stockflux-components/src/components/ScrollWrapperY/ScrollWrapperY.css

```diff
@@ -0,0 +1,14 @@
+.scroll-wrapper-y {
+  flex: 1 1 auto;
+  position: relative;
+  overflow-y: hidden;
+  overflow-x: hidden;
+  -webkit-transition: 0.6s all;
+  transition: 0.6s all;
+  height: inherit;
+}
+
+.scroll-padding-right:hover {
+  padding-right: 13px !important;
```

Comment:
Does this need !important?

Reply:
Needed to be at some point, but not anymore, thanks for noticing! Fixed -> 08cff915e0c7c162c3c71c72e9adb0e02ba1de6c